tests/tel2puml/test_tel2puml_types.py

```diff
@@ -0,0 +1,86 @@
+"""Tests for the pydantic models within tel2puml_types.py"""
+
+import tempfile
+
+import pytest
+from pydantic import ValidationError
+
+from tel2puml.tel2puml_types import OtelToPVArgs, PvToPumlArgs
+
+
+@pytest.mark.parametrize("command", ["otel2pv", "otel2puml"])
+def test_otel_to_pv_args(command: str) -> None:
```

Comment:
Sorry didn't look at this as thoroughly as I should have. Can we have some negative tests for types i.e. 

- check something that isn't a string/file path in for config file
- check integers for the boolean (do we want to make sure they can only set booleans with a boolean valueand not say an int of 0 or 1 just to reduce user error - pretty sure PyDantic will coerce 0 and 1 to bool so you can make it strict so it doesn't do that)